AppManager/WebSocket.cpp

```
@@ -0,0 +1,270 @@
+#include "WebSocket.h"
+
+#include <cstdlib>
+#include <iostream>
+
+namespace WPEFramework {
+namespace Plugin {
+
+    ConnectionMetaData::ConnectionMetaData(int id, websocketpp::connection_hdl connectionHandle, std::string uri)
```

cppcheck (2.10-2)

[performance] Function parameter 'uri' should be passed by const reference.

(passedByValue)